components/ReusableSections/JoinThrivingSection/JoinThrivingSection.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Section from 'components/_common_/Section/Section';
+import Button from 'components/_common_/Button/Button';
+import OutboundLink from 'components/_common_/OutboundLink/OutboundLink';
+import styles from './JoinThrivingSection.css';
+
+const JoinThrivingSection = ({ className }) => (
```

Comment:
Can you delete this and use the section within `components/ReusableSections`?

Reply:
to see if I understand this properly.
Do you want me to adjust the JoinSection and get rid of the JoinThrivingSection?